docs/source/get-started/multi-framework-support.rst

```diff
@@ -0,0 +1,113 @@
+and `JAX <https://jax.readthedocs.io/en/latest/index.html>`_ 
+thanks to an integration with `Ivy <https://github.com/ivy-llc/ivy>`_. 
+
+This can be accomplished using the following functions, which are now part of the kornia api:
+
+* :code:`kornia.to_tensorflow()`
+
+* :code:`kornia.to_jax()`
+
+* :code:`kornia.to_numpy()`
+
+Here's an example of using kornia with TensorFlow:
+
+.. code:: python
+
+    import kornia
+    import tensorflow as tf
+
+    tf_kornia = kornia.to_tensorflow()
+
+    rgb_image = tf.random.normal((1, 3, 224, 224))
+    gray_image = tf_kornia.color.rgb_to_grayscale(rgb_image)
+
+So what's happening here? Let's break it down.
+
+#. Transpiling kornia to TensorFlow
+
+    This line lazily transpiles everything in the kornia api to TensorFlow, and creates a new module for this transpiled version of kornia.
+    Because the transpilation happens lazily, no function or class will be transpiled until it's actually called.
+
+    .. code-block:: python
+
+        tf_kornia = kornia.to_tensorflow()
+
+#. Calling a TF kornia function
+
+    We can now call any kornia function (or class) with TF arguments. However, this function will be very slow relative to
+    the original function - as the function is being transpiled during this step.
+
+    .. code-block:: python
+
+        rgb_image = tf.random.normal((1, 3, 224, 224))
+        gray_image = tf_kornia.color.rgb_to_grayscale(rgb_image)  # slow
+
+#. Subsequent function calls
+
+    The good news is any calls of the function after the initial call will be much faster, as it has already been transpiled, 
+    and should approximately match the speed of the original kornia function.
+
+    .. code-block:: python
+
+        gray_image = tf_kornia.color.rgb_to_grayscale(rgb_image)  # fast
+
+#. Transpilations in different Python sessions
+
+    You may be wondering if you'll have to wait for these long initial transpilations to take place each time you start a
+    new Python session? The good news is that when a transpilation occurs, Ivy will save the generated source code in the
+    local directory, so if the same transpilation is ever attempted again from within the same directory, it will be
+    immediately retrieved and used.
+
+
+Kornia can be used with JAX and NumPy in the same way:
+
+.. code:: python
+
+    import kornia
+    import numpy as np
+
+    np_kornia = kornia.to_numpy()
+
+    rgb_image = np.random.normal(size=(1, 3, 224, 224))
+    gray_image = np_kornia.color.rgb_to_grayscale(rgb_image)
+
+
+.. code:: python
+
+    import kornia
+    import jax
+
+    jax_kornia = kornia.to_jax()
+
+    rgb_image = jax.random.normal(jax.random.key(42), shape=(1, 3, 224, 224))
+    gray_image = jax_kornia.color.rgb_to_grayscale(rgb_image)
+
+
+Limitations
+-----------
+
+* Converting Kornia to TensorFlow or JAX works for both functions and classes, but converting to NumPy only supports functions.
```

Comment:
I'm not sure if this qualifies as a limitation since this behavior is expected by default, right?

However, there are a few actual limitations we can point out, including:

- In-place updates not functioning as expected
- Issues with handling views
- Inability to transpile *certain* stateful classes such as optimizers (`torch.optim.Adam`, etc.), trainers, and data loaders
- Inability to transpile custom kernels like `FlashAttention`